docs/api/geo_datasets.csv

```diff
@@ -1,5 +1,6 @@
 Dataset,Type,Source,License,Size (px),Resolution (m)
 `Aboveground Woody Biomass`_,Masks,"Landsat, LiDAR","CC-BY-4.0","40,000x40,000",30
+`Airphen`_,Imagery,Airphen,-,"1,280x960",0.047--0.09
```

Comment:
License is up to the user who publishes their imagery.

Images are usually stitched together to create larger mosaics.

Resolution depends on customizable focal length and drone height, above values are for an altitude of 100 m.